examples/protonet/README.md

```diff
@@ -0,0 +1,95 @@
+# Prototypical Networks for Few-shot Learning
+
+### 1. Description
+
+This example demonstrates how to implement [Prototypical Networks for Few-shot Learning](https://github.com/jakesnell/prototypical-networks) within the PyKale pipeline using PyTorch-Lightning. Below, we offer guidance on how to use ProtoNet effectively.
+
+## 2. Usage
+
+### Datasets
+
+This data loader can be used on several few-shot learning datasets, such as mini-ImageNet, tiered-ImageNet and Omniglot, etc. (For some datasets, they need to be refactored as follows.)
+
+<pre>
+└── root
+    ├── train
+    |   ├── class 1
+    |   |   ├── image 1
+    |   |   ├── image 2
+    |   |   └── ...
+    |   ├── class 2
+    |   |   ├── image 1
+    |   |   ├── image 2
+    |   |   └── ...
+    |   └── ...
+    ├── val
+    |   ├── class m
+    |   |   ├── image 1
+    |   |   ├── image 2
+    |   |   └──...
+    |   ├── class m+1
+    |   |   ├── image 1
+    |   |   ├── image 2
+    |   |   └──...
+    |   └── ...
+    └── test (optional)
+        ├── class n
+        |   ├── image 1
+        |   ├── image 2
+        |   └── ...
+        ├── class n+1
+        |   ├── image 1
+        |   ├── image 2
+        |   └── ...
+        └── ...
+</pre>
+
+### Example Running
+
+#### Single GPU training:
+
+Example - Training ResNet18 on Omniglot in a 5-way-5-shot Protocol
```

Comment:
It will be good to explain what `way` and `shot` means before this (as well as their symbols).

Reply:
Added a description about what are N-Way and K-Shot.